Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM 169942020521.dkr.ecr.eu-west-1.amazonaws.com/base/golang:1.15-alpine-builder AS builder
+
+FROM 169942020521.dkr.ecr.eu-west-1.amazonaws.com/base/golang:1.15-alpine-runtime
+
+COPY --from=builder /build/assets ./assets
```

Comment:
I think it is simpler to do `ADD assets ./assets`. This way there is no need to create alias for builder.

Reply:
ok sure - I wasn't sure if we wanted to avoid touching the host filesystem outside of the builder base image as we have the `COPY . ./` already defined there, but agreed that it is much more simplistic.